scores.py

```diff
@@ -0,0 +1,18 @@
+from nameko.rpc import rpc, RpcProxy
+
+
+class ScoreService(object):
+    name = 'score_service'
+
+    player_rpc = RpcProxy('players_service')
+
+    @rpc
+    def leaderboard(self):
+        players = self.player_rpc.get_players()
+        sorted_players = sorted(players, key=lambda player: player.score, reverse=True)
+        return [(p.name, p.score) for p in players]
+
+    @rpc
```

Comment:
This maybe should be eventHandler like in here: https://nameko.readthedocs.org/en/stable/built_in_extensions.html?highlight=events#events-pub-sub .. Then we'll be able to do thinks like ```self.dispatch("battle_finished", [battle_id, winner_pokemon_uuid, looser_pokemon_uuid])```